testdata/baselines/reference/submoduleAccepted/conformance/typedefMultipleTypeParameters.errors.txt.diff

```diff
@@ -3,7 +3,7 @@
 @@= skipped -0, +0 lines =@@
 -a.js(12,23): error TS2344: Type '{ a: number; }' does not satisfy the constraint '{ a: number; b: string; }'.
 -  Property 'b' is missing in type '{ a: number; }' but required in type '{ a: number; b: string; }'.
-+a.js(12,23): error TS2741: Property 'b' is missing in type '{ a: number; }' but required in type '{ a: number; b: string; }'.
++a.js(12,23): error TS2741: Property 'b' is missing in type '{ a: number; }' but required in type 'Everything<{ a: number; }, undefined, { c: 1; d: 1; }, number, string>'.
```

Comment:
This new message seems wrong to me, but maybe I'm misunderstanding something? I thought the error was supposed to be actually between types `{ a: number }` and the constraint of the corresponding type parameter, `{ a: number, b: string }`. To me it seems that now we're somehow comparing the type argument to the whole instantiated type, which doesn't look right.

Reply:
Definitely wrong--well, it was previously not what Strada had, but even wronger than before.

Reply:
good catch; this fix exposed the fact that the host of typedef constraints was the typedef, not the type parameter it constrained